tests/conftest.py

```
@@ -456,20 +480,40 @@ def mock_submit(coro, loop=None):
+        if "AsyncMockMixin" in coro.__qualname__:
+            # AsyncMock coroutines need to be consumed by sending None to them
+            try:
+                # Send None to start and complete the coroutine
+                coro.send(None)
+            except StopIteration as e:
+                # Coroutine completed successfully
+                result = e.value
+                future = Future()
+                future.set_result(result)
+                return future
+            except Exception as e:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)